applications/nrf_desktop/src/modules/Kconfig.usb_state

```diff
@@ -178,6 +178,22 @@ config USBD_HID_IN_BUF_COUNT
 	  nRF Desktop queues HID reports at the source. There is no need to use
 	  multiple buffers in the IN pool per HID instance.
 
+config DESKTOP_USB_STACK_NEXT_DISABLE_ON_VBUS_REMOVAL
+	bool
+	default y if SOC_SERIES_NRF54HX
```

Comment:
`select REBOOT`

Reply:
btw. Does reboot already work on nRF54H? IIRC there used to be a related issue before

Reply:
Sys reboot is also used in the SUIT DFU using CC and SMP. It has to work

Reply:
```
static void reboot_request_handler(struct k_work *work)
{
	LOG_INF("Handle reset request now");

	if (flash_area) {
		terminate_dfu();
	}

	LOG_PANIC();

#if CONFIG_SUIT
	if ((update_candidate.mem != 0) && (update_candidate.size != 0) &&
	    (cur_offset == img_length) && !slot_was_used_by_other_transport) {
		LOG_INF("Configuration channel reboot request triggered SUIT update");
		int ret = suit_trigger_update(&update_candidate, UPDATE_CANDIDATE_CNT);

		LOG_INF("SUIT update triggered with status: %d", ret);
	}
#endif

	sys_reboot(SYS_REBOOT_WARM);

	if (!k_work_delayable_is_pending(&background_erase)) {
		config_channel_dfu_lock_release();
	}
}

```

Reply:
Just double checking, since I recall some related issues